lnrpc/routerrpc/router.proto

```diff
@@ -490,6 +490,120 @@ message BuildRouteResponse {
     lnrpc.Route route = 1;
 }
 
+
+message SubscribeHtlcEventsRequest{}
+
+message HtlcEvent{
+    /**
+    IncomingChannel is the channel that the incoming htlc arrived at our node
+    on. This value is zero for sends.
+    */
+    uint64 incoming_channel = 1 [json_name="incoming_channel"];
+
+    /**
+    Outgoing channel is the channel that the outgoing htlc left our node on.
+    This value is zero for receives.
+    */
+    uint64 outgoing_channel = 2 [json_name="outgoing_channel"];
```

Comment:
@guggero has a PR open to remove all use of `json_name`, i think we can leave them out. iirc it doesn't do anything if the names match the field name but i could be mistaken

Reply:
sgtm, will remove